projects.md

```diff
@@ -0,0 +1,45 @@
+# Projects
+
+## Sales leads & internal proposals
```

Comment:
Think we do title case

Reply:
Check out my issue comment on lead vs prospect vs opportunity. Feels like it could inform our process a bit, given that there's the distinction between _leads_ that can be totally imaginary or conceptual, vs _prospect_ where the potential client is receptive. (_opportunity_ being the final thing that our "proposal" gets us to?)

Reply:
Thanks for the review and @patcon for the research. See revised version.